src/Makefile.in

```diff
@@ -0,0 +1,32 @@
+CC ?= gcc
+LUAJIT_FLAGS = $(shell pkg-config --cflags --libs luajit)
+LUAJIT_VER = $(shell pkg-config --variable=abiver luajit)
+LUASRC = $(shell find . -regex '[^\#]*\.lua' -printf '%P ')
+
+
+BINDIR = $(DESTDIR)/@DYZBINDIR@
+LUADIR = $(DESTDIR)/@DYZLUADIR@
+
+dyz: dyz.c config.h
+	$(CC)  -o $@ $? $(LUAJIT_FLAGS) -Wl,--no-as-needed -Wl,-E
+
+install-bin: dyz
+	@mkdir -p $(BINDIR)
```

Comment:
I don't see any rationale there for not having a slash between DESTDIR and BINDIR.
I think the only reason for that is that configure scripts based on autoconf enforce the PREFIX or BINDIR to be always an absolute path. So its assumed that BINDIR will start with a slash.
And this may not be the case if you don't use an autoconf based script